app/src/main/java/android/coding/interview/makeitawesome/fragment/PicturesFragment.java

```diff
@@ -25,13 +46,70 @@ public static Fragment newInstance() {
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        RecyclerView rv = (RecyclerView) inflater.inflate(R.layout.pictures_list_fragment, container, false);
+        /*
+        RecyclerView rv = (RecyclerView) inflater.inflate(R.layout.album_list_fragment, container, false);
         setupRecyclerView(rv);
+        */
+        rv = (ListView) inflater.inflate(R.layout.album_list_fragment, container, false);
+        DownloadFilesTask d = new DownloadFilesTask();
+        d.execute();
         return rv;
     }
 
+    private void drawList(){
+
+    }
     private void setupRecyclerView(RecyclerView recyclerView) {
         recyclerView.setLayoutManager(new LinearLayoutManager(recyclerView.getContext()));
         recyclerView.setAdapter(new PhotosAdapter());
     }
+
+    private class DownloadFilesTask extends AsyncTask<URL, Integer, List<Album>> {
+        protected List<Album> doInBackground(URL... urls) {
+            RestClient restClient = RestClient.getInstance();
+            restClient.getPhotos();
+            List<Album> l = restClient.getPhotos();
```

Comment:
I've been a while wondering wtf was that "one" 1. I've just realized it is a lowercase L.